doc/manuals/sv.po

```diff
@@ -0,0 +1,3867 @@
+#: src/acl/external/time_quota/ext_time_quota_acl.8:253
+#: src/acl/external/unix_group/ext_unix_group_acl.8:113
+#: src/auth/basic/getpwnam/basic_getpwnam_auth.8:119
+#: src/auth/basic/LDAP/basic_ldap_auth.8:349
+#: src/auth/basic/NCSA/basic_ncsa_auth.8:101
+#: src/auth/basic/PAM/basic_pam_auth.8:126
+#: src/auth/basic/RADIUS/basic_radius_auth.8:137
+#: src/auth/basic/SASL/basic_sasl_auth.8:122
+#: src/auth/basic/SSPI/basic_sspi_auth.8:170
+#: src/auth/digest/file/digest_file_auth.8:107
+#: src/auth/negotiate/kerberos/negotiate_kerberos_auth.8:142
+#: src/auth/ntlm/SSPI/ntlm_sspi_auth.8:137
+#: src/security/cert_generators/file/security_file_certgen.8.in:182
+#: src/squid.8.in:281
+msgid "The Squid FAQ wiki"
+msgstr "Squid FAQ-wiki"
+
+#. type: Plain text
+#: src/acl/external/AD_group/ext_ad_group_acl.8:276
+#: src/acl/external/eDirectory_userip/ext_edirectory_userip_acl.8:241
+#: src/acl/external/file_userip/ext_file_userip_acl.8:119
+#: src/acl/external/kerberos_ldap_group/ext_kerberos_ldap_group_acl.8:272
+#: src/acl/external/LDAP_group/ext_ldap_group_acl.8:290
+#: src/acl/external/LM_group/ext_lm_group_acl.8:208
+#: src/acl/external/session/ext_session_acl.8:138
+#: src/acl/external/time_quota/ext_time_quota_acl.8:256
+#: src/acl/external/unix_group/ext_unix_group_acl.8:116
+#: src/auth/basic/getpwnam/basic_getpwnam_auth.8:122
+#: src/auth/basic/LDAP/basic_ldap_auth.8:352
+#: src/auth/basic/NCSA/basic_ncsa_auth.8:104
+#: src/auth/basic/PAM/basic_pam_auth.8:129
+#: src/auth/basic/RADIUS/basic_radius_auth.8:140
+#: src/auth/basic/SASL/basic_sasl_auth.8:125
+#: src/auth/basic/SSPI/basic_sspi_auth.8:173
+#: src/auth/digest/file/digest_file_auth.8:110
+#: src/auth/negotiate/kerberos/negotiate_kerberos_auth.8:145
+#: src/auth/ntlm/SSPI/ntlm_sspi_auth.8:140
+#: src/security/cert_generators/file/security_file_certgen.8.in:185
+#: src/squid.8.in:284
+msgid "The Squid Configuration Manual"
+msgstr "Konfigurationshandboken för Squid"
+
+#. type: Plain text
+#: src/acl/external/eDirectory_userip/ext_edirectory_userip_acl.8:5
+msgid "ext_edirectory_userip_acl - Squid eDirectory IP Lookup Helper"
+msgstr "ext_edirectory_userip_acl – Squid eDirectory IP-uppslagshjälp"
+
+#. type: Plain text
+#: src/acl/external/eDirectory_userip/ext_edirectory_userip_acl.8:15
+#: tools/squidclient/squidclient.1:22
+msgid "host"
+msgstr "värd"
+
+#. type: Plain text
+#: src/acl/external/eDirectory_userip/ext_edirectory_userip_acl.8:17
+#: src/acl/external/LDAP_group/ext_ldap_group_acl.8:20
+#: src/auth/basic/LDAP/basic_ldap_auth.8:18
+#: src/auth/basic/LDAP/basic_ldap_auth.8:33
+#: src/auth/basic/RADIUS/basic_radius_auth.8:16 src/squid.8.in:15
+#: tools/squidclient/squidclient.1:26
+msgid "port"
+msgstr "port"
+
+#. type: Plain text
+#: src/acl/external/eDirectory_userip/ext_edirectory_userip_acl.8:19
+msgid "LDAP version"
+msgstr "LDAP-version"
+
+#. type: Plain text
+#: src/acl/external/eDirectory_userip/ext_edirectory_userip_acl.8:21
+msgid "basedn"
+msgstr "baserad på"
```

Comment:
This word comes from syntax documentation. Please keep it a single word to keep it clear that the value is a single item. If it cannot be translated into a single word that makes sense, leave it unchanged.